_posts/2024-03-19-FOSSRIT-bug-fix.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+
+# Uncomment only one of the below categories
+categories: 
+- Bug Fix
+#- Contribution
+
+
+# Enter your name below
+author: Joshua Talbot
+---
+
+## Introduction to the community
+
+The project I chose was [FOSSRIT website](https://fossrit.github.io/), which Adrien introduced me to during FOSS Hours early in the semester. I initially was looking at contributing to Godot, however getting used to the file structure of godot proved to be difficult.
+
+
+My Comm Arch experiences did not cover this project, however what we covered in the three presentations showed me that the FOSSRIT website would be good to work on. Before Adrien and I started contributing to the project again, the previous commit was from October 2022, where Adrien and others were contributing to the site in their free time. This website is more of a student-led hobby project with the support of the faculty interested in Open Source at RIT, but the site needed some changes after not being updated in two years.
+
+
+As the only active maintainer at the time, Adrien has been open to answering any questions I have and giving input when applicable to my changes. We talked about how Jekyll (which is how to build the site locally) works and what could be added/changed on the site.
+
+
+## The issue
+To decide which issue I was going to tackle, I visited the github issues page while talking to Adrien and I just read what the issues were trying to resolve. I was interested in [Issue #188](https://github.com/FOSSRIT/fossrit.github.io/issues/188) of the project, which was titled _"point open at RIT page to the current open@RIT site"_.
+
+
+I first went about pursuing the issue by changing the contents of the [Open@RIT page](https://fossrit.github.io/open-rit/) on the website, however Adrien liked the snippets that were talking about Open@RIT's history, so I left that for now... (That is my plan for my contribution!). The issue was that the Open@RIT page was really a placeholder until the official [Open@RIT](https://openr.it/) page was created! There is a history as to why Open@RIT has its own separate website from RIT, which I won't be covering in my contribution content but the plan is to cover that topic in the slides.
```

Comment:
"There is a history" im not sure how much history there is there tbh, unless you were able to find more than me.